CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+
+- When you create an enhancement suggestion, populate the [issue template](ISSUE_TEMPLATE.md) and provide the steps to implement the enhancement .
+- Review the[How Do I Submit A (Good) Enhancement Suggestion?](#how-do-i-submit-a-good-enhancement-suggestion) section.
+
+#### How Do I Submit Enhancement Suggestions?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/).
+
+1. Create an issue and provide the following information:
+
+- A self-explanatory title.
+- Detailed instructions to implement the enhancement and to understand the enhancement suggestion.
+- Examples to demonstrate the steps. Include links to files or reusable snippets that are used in the examples. For providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+- Describe the current functionality and the expected functionality with enhancement and provide the reason.
+- Explain the benefits of the enhancement.
+
+### Pull Request Process
+
+You can contribute to this project by creating a pull request in the repository.
+
+1. Fork the repository into your Github account.
+2. Clone the fork locally.
+3. Create a git branch.
+4. Setup the development environment and install the required dependencies. For more information, see [Setting up a Development Environment](README.md#setting-up-a-development-environment).
+5. Make the changes and commit the changes to your new branch. Ensure that the code is clear and comprehensible.
+6. Run the local unit tests by following the instruction in the [Running Unit Tests](README.md#running-unit-tests) section. If you add new features or components that require additional unit tests, ensure that the code is tested and verified with appropriate unit tests.
+7. Create a Pull Request(PR).
+8. Incorporate the review comments provided by the repository owner, if any,  and get approval.
+The repository owner merges all approved PRs to the repository.
+
+### Elastic Path Code of Conduct
+
+Elastic Path suggests following code of conduct in all communications. For example:
+
+* Use welcoming and inclusive language
+* Respect others viewpoints and experiences
+* Accept constructive criticism
+
+The contributors must follow the code of conduct in the project and public spaces
+when they represent the project or the community. For examples, when you use an official project e-mail
+address, official social media account, or
+represent the community at an online or offline event.
+
+### Rights of the Repository Owners
+
+- Define, implement, and assure code of conduct and take action if anyone breaches the standards.
+- Remove, edit, or reject comments, commits, code, wiki edits, issues, or  other contributions that does not aligned to code of conduct.
+- Restrict the access of contributors temporarily or permanently when the code of conduct is violated.
+-Define and clarify the representation of a project further.
```

Comment:
Isn't a space needed between the hyphen and the beginning of the sentence "Define"?